src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,70 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
```

Comment:
>All [](start = 28, length = 3)

We don't have an existing helper like that somewhere in a shared location?

Reply:
I am not aware of it, but maybe @CyrusNajmabadi knows it.

Reply:
> I am not aware of it

Have you performed a search for a method like that in this project?

---
In reply to: [530465899](https://github.com/dotnet/roslyn/pull/49495#discussion_r530465899) [](ancestors = 530465899)

Reply:
And in the shared Microsoft.CodeAnalysis library.

---
In reply to: [530479279](https://github.com/dotnet/roslyn/pull/49495#discussion_r530479279) [](ancestors = 530479279,530465899)

Reply:
@AlekseyTs I definitely searched such an utility. I now made a new searchand didn't find anything.
I can see other files using the All Linq extension method (which I initially used but was asked to remove).
If you prefer me to move this utility method elsewhere, pls let me know.

Reply:
> I definitely searched such an utility. I now made a new search and didn't find anything.

Thanks for confirming this. I asked specifically because "I am not aware of it" doesn't always mean "I did a search and couldn't find".

---
In reply to: [530997839](https://github.com/dotnet/roslyn/pull/49495#discussion_r530997839) [](ancestors = 530997839)

Reply:
> If you prefer me to move this utility method elsewhere, pls let me know.

I think we should move it into the SyntaxList type as an instance method next to "Any" overloads. Should keep it internal for now though.

---
In reply to: [531150389](https://github.com/dotnet/roslyn/pull/49495#discussion_r531150389) [](ancestors = 531150389,530997839)